portfolio/src/main/webapp/tasksApi.js

```diff
@@ -0,0 +1,204 @@
+var GoogleAuth;
+
+// Scope for read access to Tasks API
+const SCOPE_READ_TASKS = 'https://www.googleapis.com/auth/tasks.readonly';
```

Comment:
James left a comment in my PR regarding these variable names. To be consistent we should move TASKS to be a prefix rather than a suffix: SCOPE_TASKS_READ, TASKS_CLIENT_ID, TASKS_API_KEY etc.

Reply:
Done.